apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/ListRowsOptions.cs

```diff
@@ -40,13 +40,26 @@ public sealed class ListRowsOptions
         /// </summary>
         public ulong? StartIndex { get; set; }
 
-        internal void ModifyRequest(ListRequest request)
+        /// <summary>
+        /// Whether to use 64-bit integers for timestamps, instead of
+        /// floating point, in order to preserve precision. If not set, this is effectively true.
+        /// </summary>
+        public bool? UseInt64Timestamp { get; set; }
+
+        // This doesn't follow the usual pattern, as we want to modify the request even if options is null.
+        internal static void ModifyRequest(ListRowsOptions options, ListRequest request)
+        {
+            options?.ModifyRequest(request);
+            request.FormatOptionsUseInt64Timestamp = options?.UseInt64Timestamp ?? true;
+        }
+
+        private void ModifyRequest(ListRequest request)
         {
             if (PageToken != null && StartIndex != null)
             {
                 throw new ArgumentException($"Cannot specify both {nameof(PageToken)} and {nameof(StartIndex)}");
             }
-
+            request.FormatOptionsUseInt64Timestamp = true;
```

Comment:
This is not done on the GetQueryResultsOptions, and iit's not needed here either I think.

Reply:
Whoops, I think this is left over from an earlier version. Removing.